src/Header/Account/Account.test.jsx

```diff
@@ -37,6 +37,14 @@ describe("Account", () => {
 		},
 	};
 
+	const idamData = {
+		display_name: "Joe Bloggs",
+		links: {
+			"Joe Bloggs": "#",
```

Comment:
Same comment as before - can we use `display_name` for the rendering and not need an extra, empty link here duplicating the user's name?